cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -58,6 +58,7 @@ spec:
         # command = "/skydns"
         - -machines=http://localhost:4001
         - -addr=0.0.0.0:53
+        - -rtimeout=1s
```

Comment:
Please file an issue, assign to yourself with the details.

On Tue, Sep 29, 2015 at 10:35 AM, Abhi Shah <notifications@github.com>
wrote:

> In cluster/addons/dns/skydns-rc.yaml.in
> <https://github.com/kubernetes/kubernetes/pull/14625#discussion_r40702079>
> :
>
> > @@ -58,6 +58,7 @@ spec:
> >          # command = "/skydns"
> >          - -machines=http://localhost:4001
> >          - -addr=0.0.0.0:53
> > +        - -rtimeout=1s
>
> We will re-adjust / remove this flag if/when we pick the latest skydns
> release.
>
> Also, hopefully, we will stop using skydns as a forwarder, making this
> flag redundant.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/14625/files#r40702079>.
>